OSX and linux builds failed.
```
  OBJCXX   qt/qt_bitcoin_qt-macdockiconhandler.o
  OBJCXX   qt/qt_bitcoin_qt-macnotificationhandler.o
In file included from qt/macdockiconhandler.mm:5:
./qt/macdockiconhandler.h:8:10: fatal error: 'QMainWindow' file not found
#include <QMainWindow>
         ^
In file included from qt/macnotificationhandler.mm:5:
./qt/macnotificationhandler.h:8:10: fatal error: 'QObject' file not found
#include <QObject>
         ^
1 error generated.
make[1]: *** [qt/qt_bitcoin_qt-macdockiconhandler.o] Error 1
make[1]: *** Waiting for unfinished jobs....
1 error generated.
make[1]: *** [qt/qt_bitcoin_qt-macnotificationhandler.o] Error 1
In file included from qt/bitcoin.cpp:9:
./qt/bitcoingui.h:14:10: fatal error: 'QLabel' file not found
#include <QLabel>
```
```
Making check in src
make[1]: Entering directory `/home/travis/build/bitcoin/bitcoin/build/bitcoin-x86_64-unknown-linux-gnu/src'
make[2]: Entering directory `/home/travis/build/bitcoin/bitcoin/build/bitcoin-x86_64-unknown-linux-gnu/src'
  CXX      qt/qt_bitcoin_qt-bitcoin.o
  CXX      qt/qt_libbitcoinqt_a-bantablemodel.o
In file included from qt/bitcoin.cpp:9:0:
qt/bitcoingui.h:14:18: fatal error: QLabel: No such file or directory
 #include <QLabel>
                  ^
compilation terminated.
make[2]: *** [qt/qt_bitcoin_qt-bitcoin.o] Error 1
make[2]: *** Waiting for unfinished jobs....
In file included from qt/bantablemodel.cpp:9:0:
qt/guiutil.h:12:23: fatal error: QHeaderView: No such file or directory
 #include <QHeaderView>
```

--- a/build-aux/m4/bitcoin_qt.m4
+++ b/build-aux/m4/bitcoin_qt.m4
@@ -399,17 +399,17 @@ AC_DEFUN([_BITCOIN_QT_FIND_LIBS_WITH_PKGCONFIG],[
     qt4_modules="QtCore QtGui QtNetwork"
     BITCOIN_QT_CHECK([
       if test x$bitcoin_qt_want_version = xqt5 || ( test x$bitcoin_qt_want_version = xauto && test x$auto_priority_version = xqt5 ); then
-        PKG_CHECK_MODULES([QT], [$qt5_modules], [QT_INCLUDES="$QT_CFLAGS"; have_qt=yes],[have_qt=no])
+        PKG_CHECK_MODULES([QT5], [$qt5_modules], [QT_INCLUDES="$QT5_CFLAGS"; QT_LIBS="$QT5_LIBS" have_qt=yes],[have_qt=no])
       elif test x$bitcoin_qt_want_version = xqt4 || ( test x$bitcoin_qt_want_version = xauto && test x$auto_priority_version = xqt4 ); then
-        PKG_CHECK_MODULES([QT], [$qt4_modules], [QT_INCLUDES="$QT_CFLAGS"; have_qt=yes], [have_qt=no])
+        PKG_CHECK_MODULES([QT4], [$qt4_modules], [QT_INCLUDES="$QT4_CFLAGS"; QT_LIBS="$QT4_LIBS" ; have_qt=yes], [have_qt=no])
       fi
 
       dnl qt version is set to 'auto' and the preferred version wasn't found. Now try the other.
       if test x$have_qt = xno && test x$bitcoin_qt_want_version = xauto; then
         if test x$auto_priority_version = xqt5; then
-          PKG_CHECK_MODULES([QT], [$qt4_modules], [QT_INCLUDES="$QT_CFLAGS"; have_qt=yes; QT_LIB_PREFIX=Qt; bitcoin_qt_got_major_vers=4], [have_qt=no])
+          PKG_CHECK_MODULES([QT4], [$qt4_modules], [QT_INCLUDES="$QT4_CFLAGS"; QT_LIBS="$QT4_LIBS" ; have_qt=yes; QT_LIB_PREFIX=Qt; bitcoin_qt_got_major_vers=4], [have_qt=no])
         else
-          PKG_CHECK_MODULES([QT], [$qt5_modules], [QT_INCLUDES="$QT_CFLAGS"; have_qt=yes; QT_LIB_PREFIX=Qt5; bitcoin_qt_got_major_vers=5], [have_qt=no])
+          PKG_CHECK_MODULES([QT5], [$qt5_modules], [QT_INCLUDES="$QT5_CFLAGS"; QT_LIBS="$QT5_LIBS" ; have_qt=yes; QT_LIB_PREFIX=Qt5; bitcoin_qt_got_major_vers=5], [have_qt=no])
         fi
       fi
       if test x$have_qt != xyes; then

